env.sh

```diff
@@ -68,6 +68,8 @@ if [[ $MODE = "prod2018" || $MODE = "phase2" || $MODE = "prod2018UL" || $MODE =
         run_cmd touch ../../.installed
         run_cmd scram b -j8
         run_cmd cd ../../../..
+
+
```

Comment:
undo modifications so that `env.sh` do not appear in the changed files list for this PR.